components/coding/profileV2/AcheivementComponent.tsx

```diff
@@ -1,62 +1,132 @@
 import React, { useEffect, useState } from "react";
 import ExpandableContainer from "../ExpandableContainer";
 import { PencilAltIcon } from "@heroicons/react/outline";
-import { IntroCourseUnit } from "../../../graphql/coding/userBadges/fetchUserBadges";
-import Link from "next/link";
+import { Data } from "../../../graphql/coding/userBadges/fetchUserBadges";
 import { User } from "../../../graphql/fetchUserProfile";
+import _ from "lodash";
```

Comment:
remove this import since we're not using it (and generally try to remove all other unused imports)